packages/zk-transfer-web/src/components/Header.tsx

```diff
@@ -17,7 +21,11 @@ const Header = () => {
                         </li>
                         <li className="flex items-center">
                             <img src="/assets/logo-ether.png" className="icons" alt="ether"/>
-                            <span>0x291F1ic810…2BgSdifB7</span>
+                            <a href="https://rinkeby.etherscan.io" 
```

Comment:
this should be a dynamically generated link that includes the Ethereum address. For example, for the address `0x18956768De7DB3601a064aaa45c46f4c719DB934` it should open up this link:
```
https://rinkeby.etherscan.io/address/0x18956768De7DB3601a064aaa45c46f4c719DB934
```

Reply:
Okay. I'll do that now.